src/css/custom.css

```
@@ -86,3 +86,18 @@
 .menu__link {
   line-height: 1.35;
 }
+
+/* Override Docusaurus alert--info blue tint applied to all <details> elements */
+details {
+  --docusaurus-details-decoration-color: var(--ifm-color-emphasis-500);
+  --ifm-alert-background-color: var(--ifm-background-surface-color);
+  --ifm-alert-background-color-highlight: rgba(255, 255, 255, 0.05);
+  --ifm-alert-border-color: var(--ifm-color-emphasis-300);
+  --ifm-alert-foreground-color: var(--ifm-font-color-base);
+  background-color: var(--ifm-background-surface-color) !important;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)